scripts/redirects.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs');
+var glob = require('glob');
+var utils = require('../lib/utils');
+
+var MASTER = 'master';
+
+hexo.extend.generator.register('blog-redirects', function () {
+  return expandRedirectObjs([
+    ['blog/2015/12/16/0.0.10-release/', 'blog/introducing-aframe/'],
+    ['blog/2015/12/16/introducing-aframe/', 'blog/introducing-aframe/'],
+    ['blog/2016/03/31/aframe-v0.2.0/', 'blog/aframe-v0.2.0/']
+  ]);
+});
+
+hexo.extend.generator.register('docs-redirects', function () {
+  var redirectObjs = [getDocRootRedirectObjs(), getPreVersionedRedirectObjs()];
+  redirectObjs.push([
+    ['docs/', 'docs/' + hexo.config.aframe_version + '/guide/'],
+    ['docs/guide/', 'docs/' + hexo.config.aframe_version + '/guide/'],
+    ['docs/components/loader.html', 'docs/components/collada-model.html'],
+    ['docs/core/animation.html', 'docs/core/animations.html'],
+    ['docs/core/assets.html', 'docs/core/asset-management-system.html'],
+    ['docs/core/mixin.html', 'docs/core/mixins.html'],
+    ['docs/core/templates.html', 'https://github.com/ngokevin/aframe-template-component'],
+    ['docs/guide/cameras-and-lights.html', 'docs/guide/building-a-basic-scene.html'],
+    ['docs/guide/entering-vr.html', 'http://mozvr.com/#start'],
+    ['docs/guide/installation.html', 'docs/guide/getting-started.html'],
```

Comment:
interesting bug:

[http://localhost:4000/docs/0.1.0/guide/installation.html](http://localhost:4000/docs/0.1.0/guide/installation.html) can be loaded. it's now [https://aframe.io/docs/guide/getting-started.html](https://aframe.io/docs/guide/getting-started.html).